src/indicator/algorithm/bollingerBands.js

```diff
@@ -19,8 +19,8 @@ export default function() {
         return d.bollingerBands;
     };
 
-    d3.rebind(bollingerBands, mergedAlgorithm, 'merge');
-    d3.rebind(bollingerBands, bollingerAlgorithm, 'windowSize', 'value', 'multiplier');
+    rebind(bollingerBands, mergedAlgorithm, 'merge');
```

Comment:
Kept the 'merge' rebind - the visual tests use this to apply multiple indicators of the same type to the input data. Is it worth removing?